pkg/build/test_test.go

```diff
@@ -39,14 +39,14 @@ const (
 	homeBuild        = "/home/build"
 )
 
-var (
-	gid1000 = uint32(1000)
-)
+const buildUser = "build"
```

Comment:
My understanding of golang declarations is shaky at best, but this could be in the rest of the const declarations above, no?

Reply:
Oh, yep, I missed those.